streaming/streaming.go

```diff
@@ -590,23 +611,38 @@ func LaunchStream(ctx context.Context, sourceProfile profiles.SourceProfile, dbL
 	}
 	fmt.Println("Successfully created stream ", datastreamCfg.StreamId)
 
+	/* Note: Retrying across an LRO poll is a workaround and not a fix, use it only after checking with the API server team.
+	 * In most cases, if a long running operation leads into a retriable failure, the server would retry internally before marking the operation as failed.
+	 */
+	updateErr := backoff.Retry(func() error { return updateStream(ctx, streamInfo, projectNumberResource, datastreamCfg, dsClient) },
```

Comment:
At the time of raising this PR, I was waiting for the initial accessor PR to be merged in (it was under review), since the 152 streams test got delayed a bit, I can do a more fuller integration with accessors now on this PR and will update the code in my next push.

Reply:
We have most of the basic stuff needed to:

Add accessor for LaunchStream
Rerfactor launchsetream and add ut.

I need to think through 2 things though:

Should we have a single accessor for laucnh stream (this involves both create and update calls)
Should we have 2 separate accessors
Should resourcemanager accessor be chained from outside or should the call to reasourcemanager accessor be made outside and only output passed to the datastream_accessor.

This is the place (making 2 API calls for 1 client and 1 API call for another client) where we probably desperately need a DI framework :(

Reply:
Resolving as discssed offline. Checking this in to unblock handover of load testing. UT refactoring would be done later on.